lib/ansible/plugins/inventory/aws_ec2.py

```diff
@@ -94,6 +94,11 @@
   # create a group for each value of the Application tag
   - key: tag.Application
     separator: ''
+
+# set individual variables with compose
+compose:
```

Comment:
Is there a reason this couldn't also be done as `hostnames: [private-ip-address, ....]`? Seems like that would be the more obvious way for people used to private/public dns-name

Reply:
It definitely can be. See https://github.com/ansible/ansible/issues/40595#issuecomment-391396902

But I don't see the harm in having an example for compose since there isn't one yet and maybe someone would for some reason want inventory_hostname and ansible_host to be different. I can add a line to hostnames too though. Maybe I'll break the example section up into a few different config examples so they make more sense? Didn't add it to hostnames since having it in both places seemed confusing.

Reply:
Ah, I see then. Yes, I think examples in sections would be good to distinguish what each set of example values is actually doing.